eng/common/scripts/ChangeLog-Operations.ps1

```diff
@@ -148,6 +148,10 @@ function Confirm-ChangeLogEntry {
       Message = ""
     }
   }
+  else {
+    # Do not stop the script on error when status object is passed as param
+    $ErrorActionPreference = 'Continue'
```

Comment:
Is this needed? In general it is not a great idea to set these preference variables conditionally like this.

Reply:
I have removed it in most recent PR and suppressed error if caller has passed an object for status